frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/common-template-validations/selectors.ts

```diff
@@ -0,0 +1,24 @@
+import { iGetIn } from '../../../../../utils/immutable';
+import { CommonTemplatesValidation } from './validation-types';
+
+export const getValidationsFromTemplates = (templates): CommonTemplatesValidation[][] => {
+  return templates.map((relevantTemplate) =>
+    JSON.parse(iGetIn(relevantTemplate, ['metadata', 'annotations', 'validations'])),
+  );
+};
+
+export const getFieldValidations = (
+  validations: CommonTemplatesValidation[][],
+  jsonPath: string,
+): CommonTemplatesValidation[] => {
+  const allTemplateValidations: CommonTemplatesValidation[] = [];
+  validations.forEach((validationArray: CommonTemplatesValidation[]) => {
+    validationArray.forEach((validation: CommonTemplatesValidation) => {
```

Comment:
this hould be one level: https://github.com/openshift/console/pull/3839/files#r363103746